queue_services/account-mailer/src/account_mailer/email_templates/account_suspended_email.html

```diff
@@ -1,9 +1,14 @@
-# Your BC Registries and Online Services account has been suspended.
+# Account suspended for {{ account_number }} - {{ account_name_with_branch }}
```

Comment:
Worked with Ethan to come up with something generic that should work for EFT and PAD.